code/modules/mining/ores_coins.dm

```diff
@@ -6,37 +6,39 @@
 	icon_state = "ore2"
 	w_type = RECYK_MISC
 	max_amount = 100
-	var/material=null
+	var/material=null //legacy
```

Comment:
Fucking remove this variable entirely. Everything related to ore management except the furnace itself is broken because you didn't bother to check where this variable is used.

You should've removed it entirely and updated usages. Not done this dumb `//legacy` and then ignored updating usages.